vocabularies/PDF.xml

```diff
@@ -99,6 +99,12 @@
             <String>Max result size for PDF documents. Used if $top has been provided and $top > ResultSizeMaximum</String>
           </Annotation>
         </Property>
+        <Property Name="IANATimezoneFormat" Type="Edm.Boolean" Nullable="false">
+          <Annotation Term="Core.Description" String="Formatting DateTimeOffset in IANA-time zone supported" />
+          <Annotation Term="Core.LongDescription">
+            <String>DateTimeOffset can be formatted in user's IANA time zone or by a defined time zone-property.</String>
+          </Annotation>
+        </Property>
```

Comment:
```suggestion
        <Property Name="IANATimestampFormat" Type="Edm.Boolean" Nullable="false">
          <Annotation Term="Core.Description" String="If this is true, the PDF format supports the value `IANA-Timestamp` in `com.sap.gateway.default.iwbep.common.v0001.TableColumn/Format/DisplayFormat`" />
          <Annotation Term="Core.LongDescription">
            <String>When a table column has `Format/DisplayFormat = 'IANA-Timestamp'`, it is formatted the in the time zone contained in the `Format/IANATimezoneProperty` or by default in the user's time zone.</String>
          </Annotation>
        </Property>
```

Reply:
Hi Heiko,

I discussed your suggestion with the team and we think it is too detailed. With the annotation we just want to announce, that we support IANA time stamp formatting on a server.
Also notice that there is at least one different OData-Implementation at SAP. I am pretty sure, that they don't use our common service for describing their document description.

Best regards,

```suggestion
  <Property Name="IANATimestampFormat" Type="Edm.Boolean" Nullable="false">
          <Annotation Term="Core.Description" String="If this is true, the PDF format supports formatting columns in IANA-time zone" />
          <Annotation Term="Core.LongDescription">
            <String>A table column can be formatted using user's IANA-time zone or an IANA-time zone from a specified column.</String>
          </Annotation>
        </Property>
```

Reply:
Hi Heiko,
meanwhile we had to introduce a third way to set the IANA timezone. Therefore, I adjusted my previous suggested change slightly.
```suggestion
        <Property Name="IANATimezoneFormat" Type="Edm.Boolean" Nullable="false">
          <Annotation Term="Core.Description" String="If this is true, the PDF format supports formatting columns in IANA-time zone" />
          <Annotation Term="Core.LongDescription">
            <String>>A table column can be formatted using a fixed IANA-time zone, user's IANA-time zone or an IANA-time zone from a specified column.</String>
          </Annotation>
        </Property>
```
Best regards,
Florian

Reply:
Two things should be made clearer:
- Only columns of type `Edm.DateTimeOffset` can be formatted in this way.
- Where is the IANA time zone taken from? "Fixed IANA time zone" and "IANA time zone from a specified column" leave that unclear.

Reply:
```suggestion
        <Property Name="IANATimezoneFormat" Type="Edm.Boolean" Nullable="false">
          <Annotation Term="Core.Description" String="If this is true, the PDF format supports formatting columns of type `Edm.DateTimeOffset` in a IANA time zone given in the document description" />
        </Property>
```
If the LongDescription does not contain additional information, let's omit it.